src/main/java/seedu/address/logic/commands/AddToDoCommand.java

```diff
@@ -14,9 +14,9 @@
  * Adds a todolist event to the toDoList.
  */
 public class AddToDoCommand extends CommandToDo {
-    public static final String COMMAND_WORD = "add todo";
+    public static final String COMMAND_WORD = "add_todo";
```

Comment:
"add todo" should be a better command than "add_todo", we wouldn't want to impose on the user to type the underscore, which feels less natural than a space. Will this cause any problem if we use space instead?

Reply:
YES. It do cause a problem if we only user one parser. because that parser take the first word to be the command word. So if only one parer, we have to use one word for command word.

Reply:
So the 2 options are:
1. Use the existing parser and keep it as add_todo. We can later add shorter aliases for all the commands, like at for add_todo, ae for add_event etc. so it would be better for users.

2. Modify existing parser to allow for 2 command words. This would be more tricky, we would need to modify the regex pattern "BASIC_COMMAND_FORMAT" in SchedulerParser to make it match both 1-word and 2-word commands.

I'm a bit conflicted between the 2 options. On one hand, "add event" feels more natural than "add_event". On the other hand, most CLI commands tend to be abbreviated (like cp for copy or rm for remove), so I feel that having a short-form of our command would be more appealing to our target user rather than a 2-word command.

So I guess I would prefer option 1. How about you guys?